adagucserverEC/CDBAdapterSQLLite.h

```diff
@@ -119,5 +119,7 @@ class CDBAdapterSQLLite : public CDBAdapter {
   int setFileString(const char *tablename, const char *file, const char *dimvalue, int dimindex, const char *filedate, GeoOptions *geoOptions);
   int setFileTimeStamp(const char *tablename, const char *file, const char *dimvalue, int dimindex, const char *filedate, GeoOptions *geoOptions);
   int addFilesToDataBase();
+  int storeLayerMetadata(const char *datasetName, const char *layerName, const char *metadataKey, const char *metadatablob);
+  CT::string getLayerMetadata(const char *datasetName, const char *layerName, const char *metadataKey);
```

Comment:
What happens if "store metadata" is turned on when SQLLite is in use?

Reply:
Nothing. 
You can test it via 
`bash runtests.sh`
`bash runtests_psql.sh`

It is backwards compatible and it will provide the same results.